jobs/transforms/topic_audits_test.py

```diff
@@ -0,0 +1,82 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for jobs.transforms.topic_audits."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+
+from core.platform import models
+from jobs import job_test_utils
+from jobs.transforms import topic_audits
+from jobs.types import audit_errors
+
+import apache_beam as beam
+
+(topic_models,) = models.Registry.import_models([models.NAMES.topic])
+
+
+class ValidateCanonicalNameMatchesNameInLowercaseTests(
+        job_test_utils.PipelinedTestBase):
+
+    NOW = datetime.datetime.utcnow()
+
+    def test_process_for_not_matching_canonical_name(self):
```

Comment:
Can you add a test when the `name` is `SOMEthing` and `canonical_name` is `something`.

Reply:
done!